config/servicemappings/compute.yaml

```diff
@@ -2391,6 +2391,16 @@ spec:
             version: v1beta1
             group: compute.cnrm.cloud.google.com
           targetField: self_link
+        - tfField: certificate_manager_certificates 
+          description: |-
+            URLs to certificate manager certificate resources that are used to authenticate connections between users and the load balancer.
+            Currently, you may specify up to 15 certificates. Certificate manager certificates do not apply when the load balancing scheme is set to INTERNAL_SELF_MANAGED.
+            sslCertificates and certificateManagerCertificates fields can not be defined together.
```

Comment:
> Does the TF resource perform some basic validation and return an error?

Yes, Terraform takes care of this validation. There is a `one_of` clause is defined in the resource definition of MMv1 and it's translated to `conflicts_with` clause in TF: `ConflictsWith: []string{"ssl_certificates"},`

> Or it will be the GCP API returning HTTP 400 errors

In fact, the field certificate_manager_certificates doesn't even exist in the API. It was added to TF for a better and more convenient UX ( and also to avoid some issues that were caused by validations related to compute resources self_link names).

Reply:
Thanks for sharing